src/main.rs

```diff
@@ -260,7 +256,8 @@ async fn messages(
     .take(2000)
     .collect::<Value>();
     let env = template_engine.acquire_env()?;
-    let ctx = context! {messages};
+    let base_url = env::var("DEPLOY_BASE_URL").ok();
+    let ctx = context! { messages => messages, base_url => base_url };
```

Comment:
Writing the key when it's the same as the variable name is redundant.

Reply:
Yes you're right, I'm not used to the field init shorthand and default to being explicit.